src/Bicep.LangServer.IntegrationTests/Completions/SnippetTemplates/res-shared-image-gallery/main.combined.bicep

```diff
@@ -0,0 +1,7 @@
+resource sharedImageGallery 'Microsoft.Compute/galleries@2020-09-30' = {
+  name: 'sharedImageGallery'
+  location: resourceGroup().location
+  properties: {
+    description: 'sharedImageGalleryDescription'

```

Comment:
please update the placeholder to correspond with the property name: `description: 'description'`